tools/archlinux_pkgbuild/mos-release/PKGBUILD

```diff
@@ -1,31 +1,36 @@
-MOS_TAG=2.17.0
+# Maintainer: Deomid Ryabkov <https://github.com/rojer>
+# Contributor: Daniel Pereira <https://github.com/kriansa>
 
 pkgname=mos
-pkgver=${MOS_TAG}
+pkgver=2.19.0
```

Comment:
Standar way of defining version is using the built-in variable, and we can avoid duplication/reassignment